ebos/eclthresholdpressure.hh

```diff
@@ -313,7 +313,7 @@ private:
                 const auto& intersection = *isIt;
 
                 // ignore boundary intersections for now (TODO?)
-                if (intersection.boundary())
+                if (!intersection.neighbor())
```

Comment:
On second thought I would prefer this to be:
```
if(intersection.boundary())
  continue; // ignore boundary intersections for now (TODO?)
else if(!neighbour) //processor boundary but not domain boundary
  continue;
```
This would make clear that the boundary treatment might change, but the processor boundary not. Probably splitting hairs here.